__init__.py

```diff
@@ -181,6 +182,7 @@ def index():
 api.add_resource(ModifyMultipleChoiceOption, API_ENDPOINT_PREFIX + 'modifymultiplechoiceoptions')
 api.add_resource(DeleteMultipleChoiceOption, API_ENDPOINT_PREFIX + 'deletemultiplechoiceoptions')
 api.add_resource(ModifyMentorQuestionFrequency, API_ENDPOINT_PREFIX + 'modifymentorquestionfrequency')
+api.add_resource(AnimELLESaveData, API_ENDPOINT_PREFIX+'animellesavedata')
```

Comment:
Keilvin wants you adjust the formatting to match the rest of ours (i.e. the spacing here: API_ENDPOINT_PREFIX + 'animellesavedata')